### ISSUE TITLE
Bugsnag Gradle Plugin using soon to be removedAPI

### ISSUE THREAD

**Issue by soehler** [bugsnag/bugsnag-android-gradle-plugin#148]
### Expected behavior
After upgrading to Android Studio 3.3 project using bugsnag latest gradle plugin should build without warnings

### Observed behavior
After upgrading the project to Gradle Plugin 3.3.0 started to receive the following warning:
```
WARNING: API 'variantOutput.getAssemble()' is obsolete and has been replaced with 'variant.getAssembleProvider()'.
It will be removed at the end of 2019.
For more information, see https://d.android.com/r/tools/task-configuration-avoidance.
To determine what is calling variantOutput.getAssemble(), use -Pandroid.debug.obsoleteApi=true on the command line to display a stack trace.
Affected Modules: app
``` 
### Steps to reproduce
Update  Android Studio 3.2 to 3.3 and start to reference Gradle Plugin 3.3.0

### Version
```
bugsnagPluginVersion = '3.6.0'
bugsnagVersion = '4.10.0'
```

### Additional information
Rolling back to classpath "com.android.tools.build:gradle:3.2.0" vanishes with the warning message

#### Can't comment on Issues?
The problem is just a warning but IMHO it should be fixed soon as it will stop working end of 2019
Info about the issue can be found here :
 https://d.android.com/r/tools/task-configuration-avoidance

**Comment by fractalwrench** [comment]
Thanks for the report @soehler - this is a duplicate of #144 which was addressed in #146, and will be released shortly.

**Comment by soehler** [comment]
Fixed on version 4.0.0
Works like a charm
Thanks

**Comment by jayce211** [comment]
I am using version 4.12.0 and still having this issue.
